org.eclipse.scanning.server/src/org/eclipse/scanning/server/servlet/ScanProcess.java

```diff
@@ -336,7 +336,7 @@ private void initializeMalcolmDevice(ScanBean bean, IPointGenerator<?> gen) thro
 	private void runScript(ScriptRequest req, Consumer<ScriptResponse<?>> cons) throws EventException, UnsupportedLanguageException, ScriptExecutionException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.eclipse.scanning.api.event.EventException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)